libraries/log/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@syntest/log",
+  "version": "0.1.0",
```

Comment:
If we want the version to be 0.1.0 when release, we might need to lower the initial version

Reply:
Fixed.